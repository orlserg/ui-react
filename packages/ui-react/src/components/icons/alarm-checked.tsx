import React from 'react';

import { SVG, IProps as IPropsSVG } from '.';

export interface IProps {
    readonly svg?: IPropsSVG
}

export const IconAlarmChecked = (props: IProps) => (
    <SVG {...props.svg}>
        <path
            fillRule="evenodd"
            d="m4.793 1.293 1.414 1.414-3.5 3.5-1.414-1.414 3.5-3.5Zm17.914 3.5-3.5-3.5-1.414 1.414 3.5 3.5 1.414-1.414ZM12 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10-4.477 10-10 10Zm0-2a8 8 0 1 0 0-16 8 8 0 0 0 0 16Zm3.293-11.707L11 12.586l-2.293-2.293-1.414 1.414L11 15.414l5.707-5.707-1.414-1.414Z"
            clipRule="evenodd"
        />
    </SVG>
);

export default IconAlarmChecked;
