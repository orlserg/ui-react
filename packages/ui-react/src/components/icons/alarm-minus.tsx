import React from 'react';

import { SVG, IProps as IPropsSVG } from '.';

export interface IProps {
    readonly svg?: IPropsSVG
}

export const IconAlarmMinus = (props: IProps) => (
    <SVG {...props.svg}>
        <path
            fillRule="evenodd"
            d="m4.793 1.293 1.414 1.414-3.5 3.5-1.414-1.414 3.5-3.5Zm17.914 3.5-3.5-3.5-1.414 1.414 3.5 3.5 1.414-1.414ZM12 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10-4.477 10-10 10Zm0-2a8 8 0 1 0 0-16 8 8 0 0 0 0 16Zm-4-9v2h8v-2H8Z"
            clipRule="evenodd"
        />
    </SVG>
);

export default IconAlarmMinus;
