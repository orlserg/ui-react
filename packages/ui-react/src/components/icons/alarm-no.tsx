import React from 'react';

import { SVG, IProps as IPropsSVG } from '.';

export interface IProps {
    readonly svg?: IPropsSVG
}

export const IconAlarmNo = (props: IProps) => (
    <SVG {...props.svg}>
        <path
            fillRule="evenodd"
            d="M2.336 3.75 1.293 4.793l1.414 1.414L3.75 5.164l.507.507A9.959 9.959 0 0 0 2 12.001c0 5.522 4.477 10 10 10a9.959 9.959 0 0 0 6.329-2.258l2.964 2.964 1.414-1.414-20-20-1.414 1.414L2.336 3.75Zm14.57 14.57L5.68 7.094A8 8 0 0 0 16.905 18.32ZM20 12a7.973 7.973 0 0 1-.642 3.144l1.5 1.5A9.956 9.956 0 0 0 22 12c0-5.523-4.477-10-10-10a9.957 9.957 0 0 0-4.644 1.142l1.5 1.5A8 8 0 0 1 20 12Zm-.793-10.707 3.5 3.5-1.414 1.414-3.5-3.5 1.414-1.414Z"
            clipRule="evenodd"
        />
    </SVG>
);

export default IconAlarmNo;
