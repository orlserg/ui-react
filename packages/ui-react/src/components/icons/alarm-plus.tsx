import React from 'react';

import { SVG, IProps as IPropsSVG } from '.';

export interface IProps {
    readonly svg?: IPropsSVG
}

export const IconAlarmPlus = (props: IProps) => (
    <SVG {...props.svg}>
        <path
            fillRule="evenodd"
            d="M6.207 2.707 4.793 1.293l-3.5 3.5 1.414 1.414 3.5-3.5Zm13-1.414 3.5 3.5-1.414 1.414-3.5-3.5 1.414-1.414ZM2 12c0 5.523 4.477 10 10 10s10-4.477 10-10S17.523 2 12 2 2 6.477 2 12Zm18 0a8 8 0 1 1-16 0 8 8 0 0 1 16 0Zm-9 1H8v-2h3V8h2v3h3v2h-3v3h-2v-3Z"
            clipRule="evenodd"
        />
    </SVG>
);

export default IconAlarmPlus;
