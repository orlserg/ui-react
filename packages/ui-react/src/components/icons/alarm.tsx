import React from 'react';

import { SVG, IProps as IPropsSVG } from '.';

export interface IProps {
    readonly svg?: IPropsSVG
}

export const IconAlarm = (props: IProps) => (
    <SVG {...props.svg}>
        <path
            fillRule="evenodd"
            d="M6.207 2.707 4.793 1.293l-3.5 3.5 1.414 1.414 3.5-3.5Zm13-1.414 3.5 3.5-1.414 1.414-3.5-3.5 1.414-1.414ZM12 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10-4.477 10-10 10Zm0-2a8 8 0 1 0 0-16 8 8 0 0 0 0 16Zm5-9h-4V6h-2v7h6v-2Z"
            clipRule="evenodd"
        />
    </SVG>
);

export default IconAlarm;
