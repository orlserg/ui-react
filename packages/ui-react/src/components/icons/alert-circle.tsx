import React from 'react';

import { SVG, IProps as IPropsSVG } from '.';

export interface IProps {
    readonly svg?: IPropsSVG
}

export const IconAlertCircle = (props: IProps) => (
    <SVG {...props.svg}>
        <path
            fillRule="evenodd"
            d="M1 12c0 6.075 4.925 11 11 11s11-4.925 11-11S18.075 1 12 1 1 5.925 1 12Zm20 0a9 9 0 1 1-18 0 9 9 0 0 1 18 0Zm-9 5.998a1 1 0 1 0 0-2 1 1 0 0 0 0 2Zm1.004-12h-2.001v8h2v-8Z"
            clipRule="evenodd"
        />
    </SVG>
);

export default IconAlertCircle;
