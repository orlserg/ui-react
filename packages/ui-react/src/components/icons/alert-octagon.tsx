import React from 'react';

import { SVG, IProps as IPropsSVG } from '.';

export interface IProps {
    readonly svg?: IPropsSVG
}

export const IconAlertOctagon = (props: IProps) => (
    <SVG {...props.svg}>
        <path
            fillRule="evenodd"
            d="M23 16.556V7.444L16.556 1H7.444L1 7.444v9.112L7.444 23h9.112L23 16.556ZM8.272 3h7.456L21 8.272v7.456L15.728 21H8.272L3 15.728V8.272L8.272 3ZM12 17.998a1 1 0 1 0 0-2 1 1 0 0 0 0 2Zm1.004-12h-2.001v8h2v-8Z"
            clipRule="evenodd"
        />
    </SVG>
);

export default IconAlertOctagon;
