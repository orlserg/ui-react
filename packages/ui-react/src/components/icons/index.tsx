import React, { SVGAttributes, ReactNode } from 'react';

export interface IProps extends SVGAttributes<SVGElement> {
    readonly children?: ReactNode,
    readonly className?: string
}

export const SVG = ({
    xmlns = 'http://www.w3.org/2000/svg',
    width = 24,
    height = 24,
    viewBox = '0 0 24 24',
    ...props
}: IProps) => (
    <svg
        xmlns={xmlns}
        width={width}
        height={height}
        viewBox={viewBox}
        {...props}
    />
);
