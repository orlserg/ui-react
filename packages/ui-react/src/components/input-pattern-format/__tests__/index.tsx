import React from 'react';
import { create } from 'react-test-renderer';

import { InputPatternFormat } from '..';
import { IconEye } from '../../icons/eye';

test('Default', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            type="text"
            format="### ## ##"
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('With ID', () => {
    const tree = create(
        <InputPatternFormat
            id="test-id"
            name="default"
            type="text"
            format="### ## ##"
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('Required', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            required={true}
            type="text"
            format="### ## ##"
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('Label', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            label="Input label"
            type="text"
            format="### ## ##"
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('Error & Message string', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            label="Input label"
            isError={true}
            elError="Message"
            type="text"
            format="### ## ##"
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('Error & Message ReactNode', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            label="Input label"
            isError={true}
            type="text"
            format="### ## ##"
            elError={(
                <div>Custom error</div>
            )}
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('Description string', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            elDescription="Description string"
            type="text"
            format="### ## ##"
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('Description ReactNode', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            type="text"
            format="### ## ##"
            elDescription={(
                <div>Description ReactNode</div>
            )}
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('elBefore & elAfter', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            type="text"
            format="### ## ##"
            elBefore={<IconEye />}
            elAfter={<IconEye />}
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('elBefore', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            type="text"
            format="### ## ##"
            elBefore={<IconEye />}
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('elAfter', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            type="text"
            format="### ## ##"
            elAfter={<IconEye />}
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('disabled', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            type="text"
            format="### ## ##"
            disabled={true}
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('disabled & readOnly', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            type="text"
            format="### ## ##"
            disabled={true}
            readOnly={true}
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('readOnly', () => {
    const tree = create(
        <InputPatternFormat
            name="default"
            type="text"
            format="### ## ##"
            readOnly={true}
        />
    );

    expect(tree.toJSON()).toMatchSnapshot();
});
