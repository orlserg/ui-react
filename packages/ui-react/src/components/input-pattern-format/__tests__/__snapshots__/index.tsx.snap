// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__field"
  >
    <input
      className="input-text__input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={0}
      type="text"
      value=""
    />
  </div>
</label>
`;

exports[`Description ReactNode 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__field"
  >
    <input
      className="input-text__input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={0}
      type="text"
      value=""
    />
  </div>
  <div>
    Description ReactNode
  </div>
</label>
`;

exports[`Description string 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__field"
  >
    <input
      className="input-text__input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={0}
      type="text"
      value=""
    />
  </div>
  <span
    className="input-text__description"
  >
    Description string
  </span>
</label>
`;

exports[`Error & Message ReactNode 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__label"
  >
    Input label
  </div>
  <div
    className="input-text__field input-text__field_is-error"
  >
    <input
      className="input-text__input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={0}
      type="text"
      value=""
    />
  </div>
  <div>
    Custom error
  </div>
</label>
`;

exports[`Error & Message string 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__label"
  >
    Input label
  </div>
  <div
    className="input-text__field input-text__field_is-error"
  >
    <input
      className="input-text__input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={0}
      type="text"
      value=""
    />
  </div>
  <span
    className="input-text__error"
  >
    Message
  </span>
</label>
`;

exports[`Label 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__label"
  >
    Input label
  </div>
  <div
    className="input-text__field"
  >
    <input
      className="input-text__input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={0}
      type="text"
      value=""
    />
  </div>
</label>
`;

exports[`Required 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__field"
  >
    <input
      className="input-text__input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={true}
      tabIndex={0}
      type="text"
      value=""
    />
  </div>
</label>
`;

exports[`With ID 1`] = `
<label
  className="input-text"
  id="test-id"
>
  <div
    className="input-text__field"
  >
    <input
      className="input-text__input"
      id="test-id-input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={0}
      type="text"
      value=""
    />
  </div>
</label>
`;

exports[`disabled & readOnly 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__field input-text__field_disabled input-text__field_read-only"
  >
    <input
      className="input-text__input"
      disabled={true}
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      readOnly={true}
      tabIndex={-1}
      type="text"
      value=""
    />
  </div>
</label>
`;

exports[`disabled 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__field input-text__field_disabled"
  >
    <input
      className="input-text__input"
      disabled={true}
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={-1}
      type="text"
      value=""
    />
  </div>
</label>
`;

exports[`elAfter 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__field input-text__field_after"
  >
    <input
      className="input-text__input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={0}
      type="text"
      value=""
    />
    <svg
      height={24}
      viewBox="0 0 24 24"
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clipRule="evenodd"
        d="M12 4c2.728 0 5.336 1.43 7.766 3.781a23.31 23.31 0 0 1 2.24 2.515c.399.516.681.923.837 1.165l.344.539-.345.539c-.154.242-.437.649-.836 1.165-.66.857-1.41 1.713-2.24 2.515C17.336 18.569 14.728 20 12 20c-2.728 0-5.336-1.43-7.766-3.781a23.312 23.312 0 0 1-2.24-2.515 17.482 17.482 0 0 1-.836-1.165L.813 12l.345-.539c.155-.242.437-.649.836-1.165.66-.857 1.41-1.713 2.24-2.515C6.664 5.431 9.272 4 12 4Zm8.422 7.517a21.32 21.32 0 0 0-2.046-2.298C16.283 7.194 14.106 6 12 6 9.895 6 7.718 7.194 5.624 9.219A21.324 21.324 0 0 0 3.216 12a21.319 21.319 0 0 0 2.408 2.781C7.718 16.806 9.895 18 12 18c2.105 0 4.283-1.194 6.376-3.219A21.314 21.314 0 0 0 20.784 12c-.11-.152-.231-.314-.362-.483ZM8 12a4 4 0 1 0 8 0 4 4 0 0 0-8 0Zm6 0a2 2 0 1 1-4 0 2 2 0 0 1 4 0Z"
        fillRule="evenodd"
      />
    </svg>
  </div>
</label>
`;

exports[`elBefore & elAfter 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__field input-text__field_before-and-after"
  >
    <svg
      height={24}
      viewBox="0 0 24 24"
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clipRule="evenodd"
        d="M12 4c2.728 0 5.336 1.43 7.766 3.781a23.31 23.31 0 0 1 2.24 2.515c.399.516.681.923.837 1.165l.344.539-.345.539c-.154.242-.437.649-.836 1.165-.66.857-1.41 1.713-2.24 2.515C17.336 18.569 14.728 20 12 20c-2.728 0-5.336-1.43-7.766-3.781a23.312 23.312 0 0 1-2.24-2.515 17.482 17.482 0 0 1-.836-1.165L.813 12l.345-.539c.155-.242.437-.649.836-1.165.66-.857 1.41-1.713 2.24-2.515C6.664 5.431 9.272 4 12 4Zm8.422 7.517a21.32 21.32 0 0 0-2.046-2.298C16.283 7.194 14.106 6 12 6 9.895 6 7.718 7.194 5.624 9.219A21.324 21.324 0 0 0 3.216 12a21.319 21.319 0 0 0 2.408 2.781C7.718 16.806 9.895 18 12 18c2.105 0 4.283-1.194 6.376-3.219A21.314 21.314 0 0 0 20.784 12c-.11-.152-.231-.314-.362-.483ZM8 12a4 4 0 1 0 8 0 4 4 0 0 0-8 0Zm6 0a2 2 0 1 1-4 0 2 2 0 0 1 4 0Z"
        fillRule="evenodd"
      />
    </svg>
    <input
      className="input-text__input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={0}
      type="text"
      value=""
    />
    <svg
      height={24}
      viewBox="0 0 24 24"
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clipRule="evenodd"
        d="M12 4c2.728 0 5.336 1.43 7.766 3.781a23.31 23.31 0 0 1 2.24 2.515c.399.516.681.923.837 1.165l.344.539-.345.539c-.154.242-.437.649-.836 1.165-.66.857-1.41 1.713-2.24 2.515C17.336 18.569 14.728 20 12 20c-2.728 0-5.336-1.43-7.766-3.781a23.312 23.312 0 0 1-2.24-2.515 17.482 17.482 0 0 1-.836-1.165L.813 12l.345-.539c.155-.242.437-.649.836-1.165.66-.857 1.41-1.713 2.24-2.515C6.664 5.431 9.272 4 12 4Zm8.422 7.517a21.32 21.32 0 0 0-2.046-2.298C16.283 7.194 14.106 6 12 6 9.895 6 7.718 7.194 5.624 9.219A21.324 21.324 0 0 0 3.216 12a21.319 21.319 0 0 0 2.408 2.781C7.718 16.806 9.895 18 12 18c2.105 0 4.283-1.194 6.376-3.219A21.314 21.314 0 0 0 20.784 12c-.11-.152-.231-.314-.362-.483ZM8 12a4 4 0 1 0 8 0 4 4 0 0 0-8 0Zm6 0a2 2 0 1 1-4 0 2 2 0 0 1 4 0Z"
        fillRule="evenodd"
      />
    </svg>
  </div>
</label>
`;

exports[`elBefore 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__field input-text__field_before"
  >
    <svg
      height={24}
      viewBox="0 0 24 24"
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clipRule="evenodd"
        d="M12 4c2.728 0 5.336 1.43 7.766 3.781a23.31 23.31 0 0 1 2.24 2.515c.399.516.681.923.837 1.165l.344.539-.345.539c-.154.242-.437.649-.836 1.165-.66.857-1.41 1.713-2.24 2.515C17.336 18.569 14.728 20 12 20c-2.728 0-5.336-1.43-7.766-3.781a23.312 23.312 0 0 1-2.24-2.515 17.482 17.482 0 0 1-.836-1.165L.813 12l.345-.539c.155-.242.437-.649.836-1.165.66-.857 1.41-1.713 2.24-2.515C6.664 5.431 9.272 4 12 4Zm8.422 7.517a21.32 21.32 0 0 0-2.046-2.298C16.283 7.194 14.106 6 12 6 9.895 6 7.718 7.194 5.624 9.219A21.324 21.324 0 0 0 3.216 12a21.319 21.319 0 0 0 2.408 2.781C7.718 16.806 9.895 18 12 18c2.105 0 4.283-1.194 6.376-3.219A21.314 21.314 0 0 0 20.784 12c-.11-.152-.231-.314-.362-.483ZM8 12a4 4 0 1 0 8 0 4 4 0 0 0-8 0Zm6 0a2 2 0 1 1-4 0 2 2 0 0 1 4 0Z"
        fillRule="evenodd"
      />
    </svg>
    <input
      className="input-text__input"
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      tabIndex={0}
      type="text"
      value=""
    />
  </div>
</label>
`;

exports[`readOnly 1`] = `
<label
  className="input-text"
>
  <div
    className="input-text__field input-text__field_read-only"
  >
    <input
      className="input-text__input"
      disabled={true}
      inputMode="numeric"
      name="default"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      readOnly={true}
      tabIndex={-1}
      type="text"
      value=""
    />
  </div>
</label>
`;
